examples/gno.land/p/samcrew/piechart/piechart.gno

```diff
@@ -0,0 +1,110 @@
+package piechart
+
+import (
+	"math"
+	"strings"
+
+	"gno.land/p/demo/svg"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/sunspirit/md"
+)
+
+// RenderPieChart creates an SVG pie chart from given values, colors, labels, and optional title.
+// It returns the SVG markup as a string, including a markdown header if a title is provided.
+func RenderPieChart(values []float64, colors []string, labels []string, title string) string {
```

Comment:
Maybe we can use an object representing a slice instead of multiple values to ensure we have the same number of values for each pie slice

```suggestion

type PieSlice struct {
  Value float64
  Color string
  Label string
}

// RenderPieChart creates an SVG pie chart from given values, colors, labels, and optional title.
// It returns the SVG markup as a string, including a markdown header if a title is provided.
func RenderPieChart(title string, slices ...PieSlice) string {
```